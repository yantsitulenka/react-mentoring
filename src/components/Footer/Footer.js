import React from 'react';
import './footer.scss';

const Footer = (props) => <footer className="footer">{props.children}</footer>;

export default Footer;
