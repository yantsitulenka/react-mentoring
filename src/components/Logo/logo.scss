@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;400;500;700&display=swap');

.logo {
  color: #f65261;
  font-size: 24px;
  font-family: 'Montserrat', sans-serif;
  font-weight: 200;
  letter-spacing: 1px;

  span {
    font-weight: 800;
  }
}
