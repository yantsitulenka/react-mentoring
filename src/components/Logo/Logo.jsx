import React from 'react';
import './logo.scss';

const Logo = () => (
  <div className="logo">
    <span>netflix</span>
    roulette
  </div>
);

export default Logo;
