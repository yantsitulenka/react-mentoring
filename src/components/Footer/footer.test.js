import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';
import Logo from '../Logo/Logo';

it('renders correctly', () => {
  const tree = renderer
    .create(<Footer><Logo /></Footer>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
