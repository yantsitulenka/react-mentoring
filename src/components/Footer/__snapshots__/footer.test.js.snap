// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="logo"
  >
    <span>
      netflix
    </span>
    roulette
  </div>
</footer>
`;
